helm/fiaas-skipper/templates/rbac.yaml

```diff
@@ -0,0 +1,66 @@
+{{- if .Values.rbac.enabled }}
+kind: ClusterRole
+{{- if semverCompare "^1.8-0" .Capabilities.KubeVersion.GitVersion }}
+apiVersion: rbac.authorization.k8s.io/v1
+{{- else }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+{{- end }}
+metadata:
+  name: fiaas-controller
+rules:
+  - apiGroups:
+      - fiaas.schibsted.io
+    resources:
+      - applications
+      - application-statuses
+      - paasbetaapplications
```

Comment:
`paasbetaapplications` and `paasbetastatuses` are in the `schibsted.io` group, not `fiaas.schibsted.io`.

Reply:
Fixed.